### PR TITLE
Feat/router

### PR DESCRIPTION
**Pull request by vmarkwardt**
Please check my styled components  AreaLink, import props

### CODE REVIEW

**Comment by vmarkwardt** [comment]
https://trello.com/b/x8xf2fjd/seekarten-lernen-vor-ergebnis

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -2,7 +2,20 @@ import React from 'react';
...
+  
+  *{
+
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  } 
+
+body {
+  margin: 0;
```
Comment (melmare) [inline src/GlobalStyle.js]: Maybe you can order your attributes by a specific order like alphabetical oder by logic.

File: src/GlobalStyle.js [hunk 2]
```diff
@@ -2,7 +2,20 @@ import React from 'react';
 import { createGlobalStyle } from 'styled-components';
 
 const StyledGlobalSyle = createGlobalStyle`
-  margin:1em;
+  
+  *{
+
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```
Comment (melmare) [inline src/GlobalStyle.js]: What is the default margin and padding in *? Different than 0?

File: src/GlobalStyle.js (hunk 1 above)
Comment (vmarkwardt) [reply src/GlobalStyle.js]: good idea

**Pull request closed without merging**